R-Project/R/fpRerF.R

```diff
@@ -0,0 +1,110 @@
+#' Packs a forest and saves modified forest to disk for use by PackPredict function
+#'
+#' Efficiently packs a forest trained with the RF option.  Two intermediate data structures are written to disk, forestPackTempFile.csv and traversalPackTempFile.csv.  The size of these data structures is proportional to a trained forest and training data respectively.  Both data structures are removed at the end of the operation.  The resulting forest is saved as forest.out.  The size of this file is similar to the size of the trained forest.
+#'
+#' @param X an n by d numeric matrix (preferable) or data frame used to train the forest. 
+#' @param Y a numeric vector of size n.  If the Y vector used to train the forest was not of type numeric then a simple call to as.numeric(Y) will suffice as input.
+#' @param csvFileName the name of a headerless csv file containing combined data and labels.
+#' @param columnWithY is the column in the headerless csv file containing class lables.
```

Comment:
may want to mention it's 0 indexed (what it is currenctly) or 1 indexed and adjust in the code with @MrAE's "fudge factor" :)

Reply:
I'm all for starting the 0 index revolution in R.  Its about time they joined the rest of the programming community.

Reply:
![image](https://user-images.githubusercontent.com/6886669/53597328-e1f74c80-3b6f-11e9-9da3-fcb1a7579a0b.png)

Reply:
resolving for now, we'll add an issue for both python and R bindings to correct there.